parsl/monitoring/db_logger.py

```diff
@@ -57,24 +121,40 @@ def get_db_logger(host='search-parsl-logging-test-2yjkk2wuoxukk2wdpiicl7mcrm.us-
     OptionalModuleMissing
 
     """
-    logger = logging.getLogger(__file__)
-    if enable_es_logging:
+    logger = logging.getLogger(logger_name)
+    if db_logger_config_object is None:
+        logger.addHandler(NullHandler())
+        return logger
+
+    if db_logger_config_object.logger_type == 'elasticsearch':
         if not _es_logging_enabled:
             raise OptionalModuleMissing(
                 ['CMRESHandler'], "Logging to ElasticSearch requires the cmreslogging module")
 
-        handler = CMRESHandler(hosts=[{'host': host,
-                                       'port': port}],
-                               use_ssl=True,
+        handler = CMRESHandler(hosts=[{'host': db_logger_config_object.host,
+                                       'port': db_logger_config_object.port}],
+                               use_ssl=db_logger_config_object.enable_ssl,
                                auth_type=CMRESHandler.AuthType.NO_AUTH,
-                               es_index_name=index_name,
+                               es_index_name=db_logger_config_object.index_name,
                                es_additional_fields={
                                    'Campaign': "test",
```

Comment:
what's this campaign?

Reply:
Not sure, it was there in the initial Elasticsearch logger example and so I left it. From what I could tell it is an identifier that can be used with Elasticsearch but it might be arbitrary or unnecessary.